src/ripple/app/ledger/LedgerTiming.cpp

```diff
@@ -128,7 +128,7 @@ bool ContinuousLedgerTiming::haveConsensus (
     }
 
     // If 80% of current proposers (plus us) agree on a set, we have consensus
-    if (((currentAgree * 100 + 100) / (currentProposers + 1)) > 80)
+    if (((currentAgree * 100 + 100) / (currentProposers + 1)) >= CONSENSUS_PCT)
```

Comment:
Actually, with your formula and currentAgree=3 and currentProposers=4, it amounts to:
((3+1) x 100 + 4) / 5 = 0.808

The rest of the constants in that source file referred to PERCENT as PCT